yapapi/runner/ctx.py

```diff
@@ -154,6 +154,9 @@ async def post(self):
 
 
 class WorkContext:
+    """Contains commands scheduled to be sent to provider.
```

Comment:
Not sure about `Contains ...`. Maybe: `An object used to build sequences of commands that are sent to providers.`?

Reply:
Or: `An object used to schedule commands to be sent to provider.`

Reply:
Ok